ext/opentelemetry-ext-jaeger/src/opentelemetry/ext/jaeger/__init__.py

```diff
@@ -0,0 +1,369 @@
+# Copyright 2018, OpenCensus Authors
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Jaeger Span Exporter for OpenTelemetry."""
+
+import logging
+import socket
+import typing
+
+from thrift.protocol import TBinaryProtocol, TCompactProtocol
+from thrift.transport import THttpClient, TTransport
+
+import opentelemetry.trace as trace_api
+from opentelemetry.ext.jaeger.gen.agent import Agent as agent
+from opentelemetry.ext.jaeger.gen.jaeger import Collector as jaeger
+from opentelemetry.sdk.trace.export import Span, SpanExporter, SpanExportResult
+
+DEFAULT_AGENT_HOST_NAME = "localhost"
+DEFAULT_AGENT_PORT = 6831
+DEFAULT_COLLECTOR_ENDPOINT = "/api/traces?format=jaeger.thrift"
+
+UDP_PACKET_MAX_LENGTH = 65000
+
+logger = logging.getLogger(__name__)
+
+
+class JaegerSpanExporter(SpanExporter):
+    """Jaeger span exporter for OpenTelemetry.
+
+    Args:
+        service_name: Service that logged an annotation in a trace.Classifier
+            when query for spans.
+        agent_host_name: (Optional) The host name of the Jaeger-Agent.
+        agent_port: (Optional) The port of the Jaeger-Agent.
+        collector_host_name: (Optional) The host name of the Jaeger-Collector
+            HTTP Thrift.
+        collector_port: (Optional) The port of the Jaeger-Collector HTTP
+            Thrift.
+        collector_endpoint: (Optional) The endpoint of the Jaeger-Collector
+            HTTP Thrift.
+        username: (Optional) The user name of the Basic Auth if authentication
+            is required.
+        password: (Optional) The password of the Basic Auth if authentication
+            is required.
+    """
+
+    def __init__(
+        self,
+        service_name: str = "my_service",
```

Comment:
Is this a sane name? It looks like it might have been copied from some example documentation.

Reply:
Just another copy & paste from OC: https://github.com/census-instrumentation/opencensus-python/blob/3f01007a05dcae84cc912a28588d3074fff4f587/contrib/opencensus-ext-jaeger/opencensus/ext/jaeger/trace_exporter/__init__.py#L79.
What do you think could be a better one?  Maybe we can also make this argument mandatory.

Reply:
Yes, please make this a mandatory argument!